client/src/components/WebTestingII.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+import App from '../App';
+import Dashboard from './Dashboard';
+import Display from './Display';
+
+describe('The App Component', () => {
+  it('Should render', () => {
+    render(<App />);
+  });
+});
+
+describe('The Dashboard Component', () => {
+  it('Should render', () => {
+    render(<Dashboard />);
+  });
+
+  it('Should increase strike by 1 when clicked', () => {
+
```

Comment:
Try to get some of the functionality (besides rendering) tested when/if you get the chance!